src/guc.c

```diff
@@ -292,6 +293,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable("timescaledb.enable_tiered_reads",
+							 "Enable tiered data reads",
+							 NULL,
```

Comment:
Most other GUCs have a more detailed explanation as third parameter. It would be good if we could add this explanation to this setting as well.

Reply:
Added a long description